server/monitor.go

```diff
@@ -2371,6 +2371,24 @@ func (s *Server) JszAccount(opts *JSzOptions) (*AccountDetail, error) {
 	return s.accountDetail(jsa, opts.Streams, opts.Consumer, opts.Config), nil
 }
 
+// helper to get cluster info from node via dummy group
+func (s *Server) raftNodeToClusterInfo(node RaftNode) *ClusterInfo {
+	if node == nil {
+		return nil
+	}
+	peers := node.Peers()
+	peerList := make([]string, len(peers))
+	for i, p := range node.Peers() {
+		peerList[i] = p.ID
```

Comment:
Should we go one layer deeper and check if the server is offline etc?

Reply:
this utility is just to convert the raftnode into a raft group and thus use s.js.clusterInfo(group).
Then this function deals with current etc...